cmd/gmnhg/main.go

```diff
@@ -83,10 +83,12 @@
 // loaded from the Hugo configuration file (config.toml, config.yaml,
 // or config.json).
 //
-// A new setting, geminiBaseURL, should be added to the Hugo
-// configuration file to ensure that RSS paths are correct. This is
-// more or less the same as Hugo's baseURL, but is separate in case
-// your Gemini site is deployed to a different server.
+// gmnhg provides a way to override these attributes by defining a
+// `gmnhg` section in the configuration file and nesting the attributes
```

Comment:
Sadly, godoc [doesn't speak proper Markdown](https://pkg.go.dev/go/doc#ToHTML). It only parses inline headings when those are single lines with no punctuation and a paragraph below, and code blocks when a paragraph is indented. pkg.go.dev seems to follow the same rules.